tasks/request_files/request_files.py

```diff
@@ -360,7 +362,8 @@ def object_exists(s3_cli: BaseClient, glacier_bucket: str, file_key: str) -> boo
     except ClientError as err:
         LOGGER.error(err)
         code = err.response['Error']['Code']
-        if code == 'NoSuchKey' or code == 'NotFound':  # Unit tests say 'NotFound', some online docs say 'NoSuchKey'
+        message = err.response['Error']['Message']
+        if message == 'NoSuchKey' or message == 'Not Found' or code == '404':  # Unit tests say 'NotFound', some online docs say 'NoSuchKey'
```

Comment:
Is `Not Found` in message supposed to have a space or should there be no space like `NotFound`?

Reply:
I am seeing "Not Found" in the error message.

`{"message": "An error occurred (404) when calling the HeadObject operation: Not Found", "timestamp": "2021-05-24T14:28:07.521580", "level": "error"}
{'Error': {'Code': '404', 'Message': 'Not Found'}`

Reply:
ok, I would change the comment to reflect that to prevent any confusion in the future.

Reply:
changed comment to 'Not Found'